chains/chains.json

```diff
@@ -0,0 +1,47 @@
+[
+  {
+    "chainId": "91b171bb158e2d3848fa23a9f1c25182fb8e20313b2c1eb49219da7a70ce90c3",
+    "assets": [
+      {
+        "assetId": 0,
+        "name": "DOT",
+        "symbol": "DOT",
+        "icon": "https://raw.githubusercontent.com/soramitsu/fearless-utils/master/chains/icons/polkadot.svg",
+        "precision": 10,
+        "isUtility": true
```

Comment:
Utility asset can be only one, so maybe we should consider making Utility asset as asset with assetId == 0?

Reply:
Good idea. Fixed